racktables/get_unused_machines.py

```diff
@@ -32,5 +32,5 @@
     obj.from_path(url_path)
     try:
         print(obj.fqdn, flush=True)
-    except:
+    except Exception:
         print(obj.common_name, flush=True)
```

Comment:
what Exception is expected here? likely that _obj_ is not defined or something. but it looks to me that the try should enclose the

```url_path = result_obj.find("a")["href"]
    obj = RacktablesObject(rt)
   obj.from_path(url_path)```
   
 and not the print. wdyt?

Reply:
https://github.com/os-autoinst/scripts/pull/395#discussion_r2099522002